scripts/remove-quant-ph/script.py

```diff
@@ -0,0 +1,18 @@
+from inspirehep.curation.search_check_do import SearchCheckDo
+
+
+class RemoveQuantPh(SearchCheckDo):
+    """Remove core from quant-ph literature records"""
+
+    query = "arxiv_eprints.categories:quant-ph core:true not _desy_bookkeeping.status:final not _desy_bookkeeping.status:printed"
```

Comment:
It has also `earlier_date` in the issue's query, not sure if we need it but without that it affects more records. Could we please double check?

Reply:
I copied the query from the issue 
https://inspirehep.net/literature?sort=mostrecent&size=250&page=1&q=arxiv_eprints.categories%3Aquant-ph%20core%3Atrue%20not%20_desy_bookkeeping.status%3Afinal%20%20not%20_desy_bookkeeping.status%3Aprinted&earliest_date=1930--2022

Reply:
I don't understand it cause the query in the `list of affected records` doesn't contain a specified date ( 2021-10-21 - 2022-11-16) range. Should I add it? Or is the query correct @michamos ?

Reply:
the query is almost correct, it's missing the earliest date part which was done as a facet in the link. The filtering by creation date based was an earlier idea that was later refined, so is not needed any longer. You can add `and de < 2023` to reproduce the effects of the earliest date facet filter.